fog/view/server/src/sharding_strategy.rs

```diff
@@ -140,4 +167,80 @@ mod epoch_sharding_strategy_tests {
 
         assert!(!should_process_block)
     }
+
+    #[test]
+    fn is_ready_to_serve_tx_outs_first_epoch_processed_block_count_is_zero_less_than_50_percent_processed_returns_true(
+    ) {
+        // The first epoch has a start block == 0.
+        const START_BLOCK: BlockIndex = 0;
+        const END_BLOCK_END_BLOCK_EXCLUSIVE: BlockIndex = 100;
```

Comment:
nit looks like this got double tagged, rest of tests cases too:
```suggestion
        const END_BLOCK_EXCLUSIVE: BlockIndex = 100;
```

Reply:
Done.

Reply:
Is this fixed in a later PR?